Chess/Cell.cs

```diff
@@ -0,0 +1,55 @@
+﻿using System;
+
+namespace Chess
+{
+    public class Cell
+    {
+        char column;
+        int row;
+
+        public Cell(char column, int row)
+        {
+            if ((column >= 'A' && column <= 'H') && (row <= 8 && row >= 1))
+            {
+                this.column = column;
+                this.row = row;
+            }
+            else
+                throw new MyNotCorrectedInputException("Input your cell again:\n");
+        }
+
+        private int GetNumberIndexOfLetter()
+        {
+            return column - 'A' + 1;
+        }
+
+        public string GetCellColour()
+        {
```

Comment:
Better return (this.GetNumberIndexOfLetter() + this.row) % 2 == 0)?"Black": "White";